src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -87,10 +87,10 @@ public String getColumnValue(final VariantContext variant, final boolean forceLo
                 throw new UserException("Cannot be missing required value for alternate_bases.AS_RAW_MQ, RAW_MQandDP or RAW_MQ.");
             }
             if (!forceLoadingFromNonAlleleSpecific && out != null) {
-                if(!out.endsWith("|0.00")) {
 // KC: we are seeing a TON of these!
+//                if (!out.endsWith("|0.00")) {
```

Comment:
commented out code

Reply:
Yeah, @kcibul commented out the logger statement, I just expanded the commenting out to take out all the functional code.